build-vitess-packages.sh

```diff
@@ -92,44 +104,49 @@ PREFIX=${PREFIX:-/usr}
 
 echo "Creating Debian Package..."
 fpm \
-   --force \
-   --input-type dir \
-   --name vitess \
-   --version "${DEB_VERSION}" \
-   --url "https://vitess.io/" \
-   --description "${DESCRIPTION}" \
-   --license "Apache License - Version 2.0, January 2004" \
-   --prefix "$PREFIX" \
-   -C "${RELEASE_DIR}" \
-   --before-install "$VTROOT/tools/preinstall.sh" \
-   --package "$(dirname "${RELEASE_DIR}")" \
-   --iteration "${SHORT_REV}" \
-   -t deb --deb-no-default-config-files
+  --force \
+  --input-type dir \
+  --name vitess \
+  --version "${DEB_VERSION}" \
+  --url "https://vitess.io/" \
+  --description "${DESCRIPTION}" \
+  --license "Apache License - Version 2.0, January 2004" \
+  --prefix "$PREFIX" \
+  -C "${RELEASE_DIR}" \
+  --before-install "$VTROOT/tools/preinstall.sh" \
+  --package "$(dirname "${RELEASE_DIR}")" \
+  --iteration "${SHORT_REV}" \
+  -t deb --deb-no-default-config-files
 
 
 echo "Creating RHEL Package..."
 fpm \
-   --force \
-   --input-type dir \
-   --name vitess \
-   --version "${VERSION}" \
-   --url "https://vitess.io/" \
-   --description "${DESCRIPTION}" \
-   --license "Apache License - Version 2.0, January 2004" \
-   --prefix "$PREFIX" \
-   -C "${RELEASE_DIR}" \
-   --before-install "$VTROOT/tools/preinstall.sh" \
-   --package "$(dirname "${RELEASE_DIR}")" \
-   --iteration "${SHORT_REV}" \
-   -t rpm
+  --force \
+  --input-type dir \
+  --name vitess \
+  --version "${VERSION}" \
+  --url "https://vitess.io/" \
+  --description "${DESCRIPTION}" \
+  --license "Apache License - Version 2.0, January 2004" \
+  --prefix "$PREFIX" \
+  -C "${RELEASE_DIR}" \
+  --before-install "$VTROOT/tools/preinstall.sh" \
+  --package "$(dirname "${RELEASE_DIR}")" \
+  --iteration "${SHORT_REV}" \
+  -t rpm
 
 
 echo "Now updating vitess-release-roster.md ...."
 cd /workspaces/vitess-releases
 echo "| $(date +%x) | @${GITHUB_USER} | [${SHORT_REV}](https://github.com/planetscale/vitess-releases/releases/tag/${SHORT_REV}) |" >> vitess-release-roster.md
```

Comment:
We may want to add this line below the DRY_RUN if statement below. This way the file isn't modified on a dry run; and no git resets will be needed.

Reply:
Good idea, done!

Reply:
LGTM